dev/mssql/latest/docker-compose.yml

```
@@ -15,4 +15,4 @@ services:
 
 networks:
   default:
-    name: sequelize-mssql-2019-network
+    name: sequelize-mssql-2022-network
```

So other docker instances can access this network without needing to update :)

```suggestion
    name: sequelize-mssql-latest-network
```